Firestore/Source/Public/FirebaseFirestore/FIRFirestore.h

```diff
@@ -261,23 +261,67 @@ NS_SWIFT_NAME(Firestore)
 
 #pragma mark - Bundles
 
+/**
+ * Loads a Firestore bundle into the local cache.
+ *
+ * @param bundleData Data from the bundle to be loaded.
+ * @return A `FIRLoadBundleTask` (`LoadBundleTask` in Swift), which allow registering observers
+ * to receive progress updates, and completion or error events.
+ */
 - (FIRLoadBundleTask *)loadBundle:(NSData *)bundleData NS_SWIFT_NAME(loadBundle(_:));
 
+/**
+ * Loads a Firestore bundle into the local cache.
+ *
+ * @param bundleData Data from the bundle to be loaded.
+ * @param completion A block to execute when the loading is in a final state, the `error` parameter
```

Comment:
I _think_ it should be `s/the loading/loading/`. However, it would be great to check this with the tech writer (I'm not a native speaker and can easily be mistaken).